plugin/src/gatsby-node.ts

```diff
@@ -91,6 +92,7 @@ async function sourceChangedNodes(
       `gatsby-source-shopify-experimental`
     ]?.lastBuildTime
   );
+  
   for (const nodeType of shopifyNodeTypes) {
     gatsbyApi
       .getNodesByType(nodeType)
```

Comment:
Should this be updated to ```.getNodesByType(`${pluginOptions.typeName}${nodeType}`)```? I'm not entirely sure how this works under the hood.

Reply:
Yeah, that makes sense to me!

One way that you can test this is working - assuming you're only using a single instance of the plugin for now - is to just run a second build without clearing the cache (in other words, just kill your `gatsby develop` process and restart it). If you got the node type names incorrect, your data from the previous build will get garbage collected. But if you got them right, then the data will still be there.